putaway_apply/security/stock_security.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file is not listed in `__openerp__.py` -> it should probably be completely removed from the PR.